lib/swagger/swagger.yaml

```diff
@@ -518,7 +518,7 @@ paths:
   /Users:
     get: 
       summary: get a list of users
-      operationId: listUsers
+      operationId: getUsersList
```

Comment:
actually, for consistency all of the GET apis which retrieve an array of objects we have named "list<Objects>".  Is there a reason for this change?

Reply:
changed it to be consistent.